### PR TITLE
redis sample update

### PR DESCRIPTION
**Pull request by salsal97**
added chroot
added kill step for native
added perf tools
make a standard testing process for perf

### CODE REVIEW

**Comment by jxyang** [comment]
How long does the test last? Do we want to move to `solutions`?

**Comment by salsal97** [comment]
It takes about 10 seconds, we could add it to solutions if that would enhance the tests. Currently it tests the redis server-client experience by spinning up a server in sgx and client on native

**Comment by jxyang** [comment]
I think it's better to move this to `solutions` and please add a README to describe the nice test harnesses you added.

**Review by mikbras — APPROVED** [review]
Look great Saloni. Thank you for your work.

File: solutions/redis/kill.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+pid=$(ps -eaf | grep redis | grep -v grep | awk '{ print $2 }')
```
Comment (vtikoo) [inline solutions/redis/kill.sh:2]:
```suggestion
pid=$(ps -eaf | grep redis-server | grep -v grep | awk '{ print $2 }')
```

Comment (salsal97) [reply solutions/redis/kill.sh]: Added this, thank you!

**Pull request merged**